dev_guide/deployments.adoc

```diff
@@ -108,9 +106,9 @@ deployments live in the cluster. Each application has different requirements for
 availability (and other considerations) during deployments. OpenShift provides
 out-of-the-box strategies to support a variety of deployment scenarios:
 
-*Recreate Strategy* [[recreate-strategy]]
+=== Recreate Strategy [[recreate-strategy]]
 
-The `*Recreate*` `*strategy*` has basic rollout behavior, and supports
+The `*recreate strategy*` has basic rollout behavior, and supports
```

Comment:
Hmm ok. I think I see what you mean. I think my reasoning was about it being just a word that didn't require caps, but if it's a proper noun then I'll put it in.